simple-fb-instant-articles.php

```diff
@@ -86,11 +84,19 @@ function add_actions() {
 			return;
 		}
 
-		if ( false !== get_query_var( $this->endpoint, false ) ) {
+		if ( false !== get_query_var( $this->endpoint, false ) && $this->is_redirectable_endpoint() ) {
 			add_action( 'template_redirect', array( $this, 'template_redirect' ) );
 		}
 	}
 
+	function is_redirectable_endpoint(){
+		if ('' === $this->endpoint || 0 == strpos($this->endpoint, '?') ){
+			return false;
```

Comment:
@whyisjake I don't think so, If a dev doesn't want an endpoint they can filter it to empty or they can attach a campaign tag to the end, in which case we wouldn't want to create a new endpoint. This works for that, right?